pkg/condition/conditions.go

```diff
@@ -51,10 +83,27 @@ func (c *condition) WaitTillPodDataStoreSynced() {
 	}
 }
 
-// TODO: Add implementation later
 func (c *condition) IsWindowsIPAMEnabled() bool {
-	// TODO: Switch using ConfigMap
-	return true
+	// Return false if configmap not present/any errors
+	vpcCniConfigMap, err := c.K8sAPI.GetConfigMap("amazon-vpc-cni")
+	if err != nil {
+		return false
+	}
+	if len(vpcCniConfigMap.Data) == 0 {
+		return false
+	}
+	if val, ok := vpcCniConfigMap.Data[enableWindowsIPAMKey]; ok {
+		enableWinIPAM, err := strconv.ParseBool(val)
+		if err != nil {
+			return false
+		}
+		if enableWinIPAM {
+			c.log.Info("Return IsWindowsIPAMEnabled = True")
```

Comment:
this is going to generate a lot of logging.

Reply:
Noted. 
Removing the log here, changes to the flag will be logged in the Reconcile function.